dev/com.ibm.ws.microprofile.reactive.messaging.kafka/resources/com/ibm/ws/microprofile/reactive/messaging/kafka/resources/ReactiveMessaging.nlsprops

```diff
@@ -63,12 +63,18 @@ kafka.library.not.present.CWMRX1006E.useraction=Package the kafka-clients JAR fi
 # A Kafka incoming connector for the {0} channel cannot be created. The error is {1}
 kafka.create.incoming.error.CWMRX1007E=CWMRX1007E: A Kafka incoming connector for the {0} channel cannot be created. The error is {1}
 kafka.create.incoming.error.CWMRX1007E.explanation=An error that occurred when the Kafka connector was set up for an incoming reactive messaging channel prevents normal operation of MicroProfile Reactive Messaging.
-kafka.create.incoming.error.CWMRX1007E.useraction=Review the error message, server message.log, and FFDC logs to identify the problem.
+kafka.create.incoming.error.CWMRX1007E.useraction=Review the error message, the messages.log file on the server, and FFDC logs to identify the problem.
 
 # A Kafka outgoing connector for the {0} channel cannot be created. The error is {1}
 kafka.create.outgoing.error.CWMRX1008E=CWMRX1008E: A Kafka outgoing connector for the {0} channel cannot be created. The error is {1}
 kafka.create.outgoing.error.CWMRX1008E.explanation=An error that occurred when the Kafka connector was set up for an outgoing reactive messaging channel prevents normal operation of MicroProfile Reactive Messaging.
-kafka.create.outgoing.error.CWMRX1008E.useraction=Review the error message, server message.log, and FFDC logs to identify the problem.
+kafka.create.outgoing.error.CWMRX1008E.useraction=Review the error message, the messages.log file on the server, and FFDC logs to identify the problem.
+
+# {0} = channel name, {1} = error message
+kafka.create.incoming.retry.CWMRX1009W=CWMRX1009W: A Kafka incoming connector for the {0} channel could not be created but this operation will be retried. The error is {1}
+kafka.create.incoming.retry.CWMRX1009W.explanation=An error occurred when the Kafka connector was set up for an incoming reactive messaging channel but the set up will be retried.
```

Comment:
Change the second "set up" to "setup" because it is a noun. Thus, change to: `was set up for an incoming reactive messaging channel but the setup will be retried.`

Reply:
Per QA ID review, provide a couple of possible or likely errors that might have occurred to cause this error. The explanation should have more information than just a restating of the error message.

Reply:
Also, avoid future tense if possible. You can achieve this by stating "when X, the setup is retried." instead of "the setup will be retried."

Reply:
> provide a couple of possible or likely errors that might have occurred to cause this error

I think I can do this. We're calling a third party API and it's throwing an exception. We don't know all the reasons that this error can occur but we can make a guess at what it might be.

The most important part of this message is informing the user that the error occurred during channel setup and which channel it occurred for. The error message included as placeholder `{1}` is the exception thrown by the API which hopefully says what's gone wrong.

> avoid future tense if possible. You can achieve this by stating "when X, the setup is retried." instead of "the setup will be retried."

Hmm, we emit this message after catching an exception from a third-party API. We wait for a short period and then retry.

I guess I could say "the setup is being retried", even though we're actually going to wait a second or so before we actually retry it? Otherwise I'm not sure how to avoid using the future tense here.

Reply:
@jantley-ibm I've made these changes, could you take a look at the latest commit on this PR? Thanks.